lib/api_client.ex

```diff
@@ -3,37 +3,37 @@ defmodule GlassFactoryApi.ApiClient do
   Client which make external HTTP Request.
   """
 
-  @access_info Application.get_env(:glass_factory_api, GlassFactoryApi.ApiClient)
-
   @type http_response() :: {atom(), HttpAdapter.response()}
 
+  @default_configuration GlassFactoryApi.Configuration.default_configuration()
+
   @doc """
-  Make a GET request using the given adapter to retrieve the given resource
+  Make a GET request using the given configuration to retrieve the given resource.
 
   ## Examples
       iex> GlassFactoryApi.ApiClient.get("members")
       {:ok , %{body: "[]", headers: [], status_code: 200}}
+
+  If no configuration is passed, it will get the default configuration which is defined in `GlassFacotryApi.Configuration.default_configuration`
```

Comment:
```suggestion
  If no configuration is passed, it will get the default configuration which is defined in `GlassFactoryApi.Configuration.default_configuration`
```